sample.env

```diff
@@ -25,4 +25,13 @@ OPENCHANNEL_NOTIFICATION_FROM="Name4 <user4@domain.com>"
 
 CHANNELMISMATCH_NOTIFICATION_TO='["Name1 <user1@domain.com>"]'
 CHANNELMISMATCH_NOTIFICATION_CC='["Name2 <user2@domain.com>","Name3 <user3@domain.com>"]'
-CHANNELMISMATCH_NOTIFICATION_FROM="Name4 <user4@domain.com>"
\ No newline at end of file
+CHANNELMISMATCH_NOTIFICATION_FROM="Name4 <user4@domain.com>"
+
+RUN_CLIGHTNING=<true>
+RUN_LND=<true>
+
+C_LISTEN_ADDRESS=<HOSTNAME:PORT>
```

Comment:
Use LISTEN_ADDRESS_CLN and also change LISTEN_ADDRESS to LISTEN_ADDRESS_LND
It's better to have it as a suffix here because it's not a property of lnd or cln but a property of the lspd working with cln/lnd